src/app/pages/wallets/my-wallets/my-wallet/pages-wallets-my-wallets-my-wallet.component.ts

```diff
@@ -8,4 +8,10 @@ import { MyWallet } from '../pages-wallets-my-wallet.model';
 })
 export class PagesWalletsMyWalletsMyWalletComponent {
   @Input() public wallet!: MyWallet;
-}
+
+  @Output('openWalletModalCallback') openWalletModalCallback = new EventEmitter<number>();
```

Comment:
I would simplify this to `@Output() public walletEdit: EventEmitter<void> = new EventEmitter();`